doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -813,7 +814,7 @@ PyUFunc_Type and PyUFuncObject
    .. c:member:: int identity
 
        Either :c:data:`PyUFunc_One`, :c:data:`PyUFunc_Zero`,
-       :c:data:`PyUFunc_None` or :c:data:`PyUFunc_AllOnes` to indicate
+       or :c:data:`PyUFunc_None` to indicate
        the identity for this operation. It is only used for a
        reduce-like call on an empty array.
```

Comment:
Perhaps out of scope, but this is quite out of date - the full list is:

https://github.com/numpy/numpy/blob/master/numpy/core/include/numpy/ufuncobject.h#L281-L311

Reply:
Fixed.